pool/example_test.go

```diff
@@ -292,3 +297,273 @@ func ExamplePool_ExchangeDagBetweenPoolNodes() {
 	//     from QmaUMRTBMoANXqpUbfARnXkw9esfz9LP2AjXRRr7YknDAT
 	//     content: {"this":true}
 }
+
+func TestExamplePool_ExchangeDagBetweenPoolAndFula(t *testing.T) {
```

Comment:
This is a `test` but in example file? if it is an example, please make sure the function name starts with `Example` and it has `// output` comments at the end. otherwise please move it to one of the test files.

Same with the remaining tests added to this file please.